docs/examples/plot_investigate_pipeline.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+Investigate a pipeline
+======================
+
+The following example shows how to look into a converted
+models and easily find errors at every step of the pipeline.
+
+.. contents::
+    :local:
+
+Create a pipeline
++++++++++++++++++
+
+We reuse the pipeline implemented in example
+`Pipelining: chaining a PCA and a logistic regression
+<https://scikit-learn.org/stable/auto_examples/compose/plot_digits_pipe.html>`_.
+There is one change because
+`ONNX-ML Imputer <https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md#ai.onnx.ml.Imputer>`_
+does not handle string type. This cannot be part of the final ONNX pipeline
+and must be removed. Look for comment starting with ``---`` below.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from sklearn import datasets
+from sklearn.decomposition import PCA
+from sklearn.linear_model import LogisticRegression
+from sklearn.pipeline import Pipeline
+from sklearn.model_selection import GridSearchCV
+
+logistic = LogisticRegression()
```

Comment:
What's the need to declare these two variables?

Reply:
Fixed.